portfolio/src/main/webapp/script.js

```diff
@@ -16,8 +16,19 @@
  * Adds a random greeting to the page.
  */
 
-async function getHelloMessage(){
+async function getQuote(){
     const response = await fetch("/data");
-    const msg = await response.text();
-    document.getElementById("msg").innerText = msg;
+    const quote = await response.text();
+    document.getElementById("quote-container").innerText = quote;
+}
+async function getMessages(){
+    const list = document.getElementById("message-list");
+    const response = await fetch("/messages");
+    const messages = await response.json();
+    
+    messages.forEach(function(item){
```

Comment:
Prefer to use arrow functions over the "function" syntax. See https://google.github.io/styleguide/jsguide.html#features-functions-arrow-functions for some rationale